target/linux/apm821xx/dts/meraki-mx60.dts

```diff
@@ -62,10 +62,30 @@
 			};
 
 			partition@100000 {
-				label = "u-boot-env";
+				compatible = "fixed-partitions";
 				reg = <0x00100000 0x00100000>;
-				compatible = "u-boot,env";
+				label = "u-boot-env";
 				read-only;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partition@0 {
+					label = "u-boot-env-main";
+					reg = <0x0 0x20000>;
+
+					nvmem-layout {
+						compatible = "u-boot,env-redundant-cunt";
```

Comment:
Wot? I beg your pardon?! "u-boot,env-redundant-**cunt**" . So, you have acquired a MX60(W), is it too much to ask for a contributor to please test his own changes, right?

No no no, sorry I'm out. I can't take this abuse any longer.

Reply:
I must apologize for that. It's not intentional.